src/stylesheets/settings/_settings-components.scss

```diff
@@ -38,6 +38,18 @@ $theme-button-border-radius: "md" !default;
 $theme-button-small-width: 6 !default;
 $theme-button-stroke-width: 2px !default;
 
+// Card
+$theme-card-header-font-family: "ui" !default;
+$theme-card-font-family: "ui" !default;
+$theme-card-margin: 2 !default;
+$theme-card-border-radius: "md" !default;
+$theme-card-border-width: 2px !default;
+$theme-card-border-color: "base-lighter" !default;
+$theme-card-bg-color: "white" !default;
+$theme-card-shadow-density: 3 !default;
+$theme-card-padding-size: 4 !default;
+$theme-card-equal-height: false !default;
```

Comment:
I wonder if this should be a modifier flag and not a setting, so this could be adjusted from case to case

Reply:
Do you think this is something better suited to a modifier class?

Reply:
yeah, exactly